spec/factories/users.rb

```diff
@@ -30,4 +30,12 @@
     end
   end
 
+  factory :system_user, parent: :user do
+    first_name "GoodCity"
+    last_name  "Team"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.